int64[10] angle_to_motor_out
